vernac/declare.ml

```diff
@@ -981,7 +981,7 @@ module ProgramDecl = struct
     let get_obligations prg = prg.prg_obligations
     let get_using prg = prg.prg_using
   end
-end
+end (* Obls_.ProgramDecl module *)
```

Comment:
such annotation seem to have the potential to get out of date and then become really confusing

Reply:
Indeed both emacs and vscode can display the matching opening module in the editor. I'm not against the comment tho.

Reply:
It helps me to follow the structure of the code but yes, there is a risk of outdating (like often with comments).

Reply:
@ejgallego: how do you see the matching opening module in emacs?

Reply:
>@ejgallego: how do you see the matching opening module in emacs?

I have `show-paren-context-when-offscreen` set to `'child-frame`. Looks like this:
![child-frame](https://github.com/coq/coq/assets/2461932/d54022dd-7bf6-4723-801e-5b2f505aae52)

It only works properly when the `module Foo =` is on the same line as `struct`, eg
~~~ocaml
module Foo =
struct
(* 1 million lines *)
end
~~~
shows only `struct` since it only shows the opening line (the docstring says it should show the previous line when the opening paren is on its own line, but I guess it doesn't work with things that aren't actually 1 char)

Reply:
Hum, you'll explain me when we meet.

In any case, I'm removing the corresponding commit.